File: eShop_CleanArch/Domain/Entities/Review.cs
using Domain.Abstract;

namespace Domain.Entities;

public class Review : Entity<Guid>
{
    public Guid ProductId { get; set; }
    public Product? Product { get; set; }
    public Guid UserId { get; set; }
    public User? User { get; set; }
    public short Rating { get; set; }
    public string? Title { get; set; }
    public string? Comment { get; set; } 
   
}

File: eShopClient/src/app/features/models/review.ts
import { UserModel } from "../../core/models/user";
import { ProductModel } from "./product";

export class ReviewModel {
	id:string= "";
	productId: string= "";
	product:ProductModel= new ProductModel();
	userId: string= "";
	user: UserModel = new UserModel();
	rating: number= 0;
	title: string= "";
	comment: string= "";
	createAt: string= "";
}